inf-clojure.el

```diff
@@ -256,15 +256,28 @@ It requires a REPL PROC for inspecting the correct type."
     inf-clojure-repl-type))
 
 (defun inf-clojure--single-linify (string)
-  "Convert a multi-line STRING in a single-line STRING."
-  (replace-regexp-in-string "[[:space:]\\|\n]+" " " string))
+  "Convert a multi-line STRING in a single-line STRING.
+It also reduces/adds redundant whitespace for readability.  Note
+that this function will transform the empty string in \" \" (it
+adds an empty space)."
+  (replace-regexp-in-string "[ \\|\n]+" " " string))
+
+(defun inf-clojure--trim-newline-right (string)
```

Comment:
Well no, that is basically the point..the function does not want to replace tabs, because tabs can be sent to the Repl (for instance for completion)